runtime-local/src/lib.py

```diff
@@ -140,6 +149,20 @@ def json_obj_to_flat_map(obj, prefix: str = "", separator: str = ".") -> dict[st
     return result
 
 
+def get_log_file_name(service_id: str) -> str:
```

Comment:
This method not only gets the log file name but it also creates intermediate directories but not the final file. We should split responsibilities into at least 2 distinct methods: `get_log_file_name` and `create_file_and_path` (names tbd)